v20.2/disaster-recovery.md

```diff
@@ -0,0 +1,333 @@
+---
+title: Disaster Recovery
+summary: Learn how about CockrachDB disaster recovery capabilities and what to do if you encounter an issue.
+toc: true
+---
+
+While CockroachDB is built to be fault-tolerant and to recover automatically, sometimes disasters happen. A _disaster_ is any event that puts your cluster at risk, and usually means your cluster is experiencing [hardware failure](#hardware-failure), [data failure](#data-corruption), or has [compromised security keys](#compromised-security-keys). Having a disaster recovery plan enables you to recover quickly, while limiting the consequences.
+
+## Hardware failure
+
+When planning to survive hardware failures, these are the minimum replication factors to apply in order to have greater resilience.
+
+{{site.data.alerts.callout_info}}
+Increasing data replication factors may have an impact on [transaction](transactions.html) performance, since quorum levels are increasing.
+{{site.data.alerts.end}}
+
+### Single-region survivability planning
+
+The table below displays the minimum replication factor needed to survive simultaneous hardware failures (e.g., disk, node, availability zone (AZ), etc.) for a single-region CockroachDB cluster:
+
+<table>
+  <thead>
+    <tr>
+      <th>Simultaneous Failure</th>
+      <th>3 nodes <br>(3 AZ/Racks)</th>
+      <th>4 nodes <br>(4 AZ/Racks)</th>
+      <th>5 nodes <br>(5 AZ/Racks)</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td style="color:#46a417"><b>Disk</b></td>
+      <td>Rep Factor = 3</td>
+      <td>Rep Factor = 3</td>
+      <td>Rep Factor = 3</td>
+    </tr>
+      <td style="color:#46a417"><b>Node</b></td>
+      <td>Rep Factor = 3</td>
+      <td>Rep Factor = 3</td>
+      <td>Rep Factor = 3</td>
+    <tr>
+      <td style="color:#46a417"><b>AZ/Rack</b></td>
+      <td>Rep Factor = 3</td>
+      <td>Rep Factor = 3</td>
+      <td>Rep Factor = 3</td>
+    </tr>
+    <tr>
+      <td style="color:#46a417"><b>2 Nodes</b></td>
+      <td>Rep Factor = 5</td>
+      <td>Rep Factor = 5</td>
+      <td>Rep Factor = 5</td>
+    </tr>
+    <tr>
+      <td style="color:#46a417"><b>AZ/Rack + Node</b></td>
+      <td>Rep Factor = 9</td>
+      <td>Rep Factor = 7</td>
+      <td>Rep Factor = 5</td>
+    </tr>
+    <tr>
+      <td style="color:#46a417"><b>2 AZ/Racks</b></td>
+      <td>Not possible</td>
+      <td>Not possible</td>
+      <td>Rep Factor = 5</td>
+    </tr>
+  </tbody>
+</table>
+
+For example, a single-region, 3-node cluster would need a minimum replication factor of 3 to survive a disk failure.
+
+### Single-region recovery
```

Comment:
I wish we could send some message about how often AZs fail--do you know if we have that information somewhere? It'd be nice to deliver the message of 'this is unlikely to happen but in case it does, make sure you're prepared'